prac_07/programming_language.py

```diff
@@ -0,0 +1,44 @@
+"""
+CP1404/CP5632 Practical - Suggested Solution
+Programming Language class with tests.
```

Comment:
probably should change the docstring so its not just copied straight from the solutions